src/main/java/com/example/psoft_22_23_project/api/AuthApi.java

```diff
@@ -58,10 +63,14 @@ public class AuthApi {
 	public ResponseEntity<?> login(@RequestBody @Valid final AuthRequest request) {
 		String clientIp = clientIPUtil.getClientIP();
 		String username = request.getUsername();
+		long start = System.currentTimeMillis();
+		String requestId = UUID.randomUUID().toString();
+
+		logger.info("[{}] Login attempt for user '{}' from IP {}", requestId,Utils.sanitize(request.getUsername()), Utils.sanitize(clientIp));
```

Comment:
Just a suggestion, its not critical. I think it would be better to log conditionally when a method is called inside the logger. This would prevent the method to be evaluated even when the log level is not low enough to show the message.

```suggestion
    if (logger.isInfoEnabled()) {
		logger.info("[{}] Login attempt for user '{}' from IP {}", requestId,Utils.sanitize(request.getUsername()), Utils.sanitize(clientIp));
    }
```

Reply:
Ok I agree